### PR TITLE
React + Typescript + Webpack + react-admin custom admin console initial package setup

### PR DESCRIPTION
**Pull request by prince3339**
- Initial admin console project setup

- `yarn run admin:dev` should build admin on watch mode
- `yarn run admin:build` should build admin for production
- `yarn dun admin:start` should start the production build correctly

### CODE REVIEW

**Review by dipta007 — APPROVED** [review]

**Review by maabed — APPROVED** [review]

File: packages/admin/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
```
Comment (Ethaan) [inline packages/admin/webpack.config.js:1]: add this file to `.gitignore`

**Comment by Ethaan** [comment]
@prince3339 check your branch names, here we are merging 2 releases, it should be `feature/DVX-437-AdminConsole` instead of `release/DVX-437-AdminConsole` i guess this was a typo

**Comment by prince3339** [comment]
> @prince3339 check your branch names, here we are merging 2 releases, it should be `feature/DVX-437-AdminConsole` instead of `release/DVX-437-AdminConsole` i guess this was a typo

@Ethaan Sorry it was a typo. Should I keep as it is? Or open a new PR with the correct branch name?

**Comment by Ethaan** [comment]
@prince3339 nah its okay, in the future there will be bots listening to `release` branch and this could be an issue, for now its OK

**Pull request merged**